jupyterhub/tests/mocking.py

```diff
@@ -93,6 +93,9 @@ def user_env(self, env):
     def _cmd_default(self):
         return [sys.executable, '-m', 'jupyterhub.tests.mocksu']
 
+    def delete_forever(self):
+        return "mocking_this_method"
```

Comment:
Use `pass` for empty base methods:

```suggestion
        pass
```

Reply:
done